core/testing/src/main/kotlin/com/google/samples/apps/nowinandroid/core/testing/repository/TestRecentSearchRepository.kt

```diff
@@ -19,14 +19,16 @@ package com.google.samples.apps.nowinandroid.core.testing.repository
 import com.google.samples.apps.nowinandroid.core.data.model.RecentSearchQuery
 import com.google.samples.apps.nowinandroid.core.data.repository.RecentSearchRepository
 import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.flow.flow
 
 class TestRecentSearchRepository : RecentSearchRepository {
 
     private val cachedRecentSearches: MutableList<RecentSearchQuery> = mutableListOf()
```

Comment:
How about using MutableSharedFlow like another TestRepository? :)

Reply:
@yongsuk44 I agree with your opinion. but because i didn't fully understand using 'MutableList' instead of "MutableSahredFlow' like other unit test code, i did not change this code.

and the primary purpose of this 'PR' is writing the test code of 'UseCase' class in domain layer. :) 

i have thought that your opinion is reasonable so far 👍

Reply:
@squart300kg 
I've determined that handling test cases reactively when utilizing this repository in the future would likely lead to a smoother process. Hence, I've left a comment to that effect. :)